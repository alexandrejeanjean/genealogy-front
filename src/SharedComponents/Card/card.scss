.card-wrapper {
  position: relative;
  .card {
    width: 12em;
    height: 12em;
    margin: 1em;
    display: flex !important;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    border: none;
    img {
      width: 6.5em;
    }
    .card-text,
    .card-subtext {
      color: #348fd9;
      min-width: 17em !important;
      max-width: 18em;
      margin: 0;
    }

    .card-text {
      line-height: 2em;
      font-weight: bolder;
      margin-top: 0.5em;
      margin-bottom: -0.5em;
    }
  }

  button.delete-btn {
    background-color: unset;
    position: absolute;
    top: -0.7em;
    right: 0;

    img {
      width: 2.2em;
    }
  }
}
