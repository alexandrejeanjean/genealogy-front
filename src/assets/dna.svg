<?xml version="1.0" encoding="iso-8859-1"?>
<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 512.001 512.001" style="enable-background:new 0 0 512.001 512.001;" xml:space="preserve">
<g>
	
		<rect x="55.658" y="389.651" transform="matrix(-0.7071 0.7071 -0.7071 -0.7071 432.6058 703.6542)" style="fill:#FFDE76;" width="29.828" height="103.543"/>
	
		<rect x="241.08" y="157.53" transform="matrix(-0.7071 0.7071 -0.7071 -0.7071 618.0268 256.002)" style="fill:#FFDE76;" width="29.828" height="196.938"/>
	
		<rect x="192.181" y="219.862" transform="matrix(-0.7071 0.7071 -0.7071 -0.7071 569.1241 374.046)" style="fill:#FFDE76;" width="29.828" height="170.061"/>
	
		<rect x="289.993" y="122.077" transform="matrix(0.7071 -0.7071 0.7071 0.7071 -57.1419 276.2622)" style="fill:#FFDE76;" width="29.828" height="170.061"/>
	
		<rect x="426.511" y="18.804" transform="matrix(0.7071 -0.7071 0.7071 0.7071 79.386 332.8052)" style="fill:#FFDE76;" width="29.828" height="103.543"/>
</g>
<g>
	<path style="fill:#FFB641;" d="M255.317,136.43l-21.092,21.092l33.239,33.239c10.757-4.287,21.909-6.873,33.834-8.35
		L255.317,136.43z"/>
	<path style="fill:#FFB641;" d="M196.918,175.825l-21.092,21.092l36.002,36.002c3.063-3.657,6.413-7.321,10.092-10.999
		c3.68-3.68,7.338-7.026,10.993-10.099L196.918,175.825z"/>
	<path style="fill:#FFB641;" d="M157.522,234.224l-21.092,21.092l45.933,45.933c1.44-11.559,4.013-22.738,8.431-33.752
		L157.522,234.224z"/>
</g>
<path style="fill:#FC7176;" d="M505.14,113.207c-44.625,44.625-96.392,43.364-146.455,42.145
	c-53.526-1.304-104.083-2.536-154.976,48.357s-49.661,101.45-48.357,154.976c1.22,50.062,2.482,101.828-42.145,146.455
	c-9.149,9.149-23.983,9.149-33.131,0c-9.149-9.149-9.149-23.982,0-33.131c30.427-30.427,29.548-66.505,28.435-112.183
	c-0.707-28.988-1.436-58.964,5.959-90.777c8.633-37.139,26.986-69.349,56.108-98.471s61.332-47.475,98.471-56.108
	c31.814-7.395,61.789-6.665,90.777-5.959c45.677,1.113,81.755,1.992,112.183-28.435c9.149-9.149,23.983-9.149,33.131,0
	C514.288,89.225,514.288,104.058,505.14,113.207z"/>
<g>
	<path style="fill:#FFB641;" d="M415.36,23.426l-21.092,21.092l39.711,39.711c3.196-9.732,8.256-18.125,16.157-26.026L415.36,23.426
		z"/>
	<path style="fill:#FFB641;" d="M84.201,433.948L44.52,394.267l-21.092,21.092l34.777,34.777
		C66.116,442.224,74.539,437.151,84.201,433.948z"/>
</g>
<path style="fill:#80C9FF;" d="M398.794,6.861c-44.625,44.625-43.364,96.392-42.145,146.455
	c1.304,53.526,2.536,104.083-48.357,154.976s-101.45,49.661-154.976,48.357c-50.062-1.22-101.828-2.482-146.455,42.145
	c-9.149,9.149-9.149,23.983,0,33.131s23.982,9.149,33.131,0c30.427-30.427,66.505-29.548,112.183-28.435
	c28.988,0.707,58.964,1.436,90.777-5.959c37.139-8.633,69.349-26.986,98.471-56.108c29.122-29.122,47.475-61.332,56.108-98.47
	c7.395-31.814,6.665-61.789,5.959-90.779c-1.113-45.677-1.992-81.755,28.435-112.183c9.149-9.149,9.149-23.983,0-33.131
	C422.777-2.287,407.943-2.287,398.794,6.861z"/>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
</svg>
