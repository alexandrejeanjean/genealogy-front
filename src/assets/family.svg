<?xml version="1.0" encoding="iso-8859-1"?>
<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 512.001 512.001" style="enable-background:new 0 0 512.001 512.001;" xml:space="preserve">
<path style="fill:#4A5058;" d="M439.809,164.995c-1.138,0-2.291-0.259-3.376-0.807L256,73.043L75.567,164.187
	c-3.696,1.868-8.208,0.384-10.076-3.313c-1.867-3.697-0.385-8.208,3.313-10.076l183.814-92.853c2.127-1.074,4.637-1.074,6.764,0
	l183.814,92.853c3.697,1.868,5.18,6.379,3.313,10.076C445.189,163.488,442.549,164.995,439.809,164.995z"/>
<rect x="20.643" y="157.491" style="fill:#FFFFFF;" width="470.71" height="278.84"/>
<path style="fill:#76C9D2;" d="M223.355,435.978c0-33.98-27.546-61.527-61.527-61.527l0,0c-33.98,0-61.527,27.546-61.527,61.527
	H223.355z"/>
<g>
	<path style="fill:#FFFFFF;" d="M161.828,393.543l-27.547,12.913c-2.541,1.191-5.458-0.663-5.458-3.47v-21.662L161.828,393.543z"/>
	<path style="fill:#FFFFFF;" d="M161.828,393.543l27.547,12.913c2.541,1.191,5.458-0.663,5.458-3.47v-21.662L161.828,393.543z"/>
</g>
<path style="opacity:0.1;enable-background:new    ;" d="M161.828,374.451L161.828,374.451c-16.866,0-32.144,6.788-43.258,17.779
	c13.019,6.516,27.709,10.188,43.258,10.188s30.239-3.673,43.258-10.188C193.972,381.24,178.694,374.451,161.828,374.451z"/>
<g>
	<circle style="fill:#FFD4B8;" cx="65.04" cy="296.881" r="17.07"/>
	<circle style="fill:#FFD4B8;" cx="258.62" cy="296.881" r="17.07"/>
	<circle style="fill:#FFD4B8;" cx="161.83" cy="296.751" r="96.79"/>
</g>
<g>
	<path style="fill:#4A5058;" d="M112.748,304.379c-4.143,0-7.5-3.358-7.5-7.5v-9.643c0-4.142,3.357-7.5,7.5-7.5s7.5,3.358,7.5,7.5
		v9.643C120.248,301.021,116.891,304.379,112.748,304.379z"/>
	<path style="fill:#4A5058;" d="M210.908,304.379c-4.143,0-7.5-3.358-7.5-7.5v-9.643c0-4.142,3.357-7.5,7.5-7.5s7.5,3.358,7.5,7.5
		v9.643C218.408,301.021,215.051,304.379,210.908,304.379z"/>
	<path style="fill:#4A5058;" d="M161.828,330.974c-5.932,0-11.662-2.485-15.721-6.819c-2.832-3.023-2.677-7.77,0.347-10.601
		c3.022-2.833,7.77-2.676,10.601,0.347c1.253,1.337,2.947,2.073,4.773,2.073c1.826,0,3.521-0.736,4.773-2.073
		c2.831-3.023,7.576-3.179,10.601-0.347c3.023,2.832,3.179,7.578,0.347,10.601C173.49,328.489,167.76,330.974,161.828,330.974z"/>
	<path style="fill:#4A5058;" d="M161.828,199.961c-53.456,0-96.791,43.335-96.791,96.791c15.044,0,30.952-16.02,32.595-48.706
		c10.305,11.541,69.265,34.371,130.584,16.028c5.157,21.898,18.067,32.678,30.403,32.678
		C258.619,243.296,215.284,199.961,161.828,199.961z"/>
</g>
<path style="opacity:0.1;fill:#513130;enable-background:new    ;" d="M176.172,381.324c-53.456,0-96.791-43.335-96.791-96.791
	c0-26.127,10.356-49.833,27.183-67.246c-25.099,17.488-41.527,46.556-41.527,79.465c0,53.456,43.335,96.791,96.791,96.791
	c27.329,0,52.008-11.33,69.608-29.544C215.766,374.917,196.719,381.324,176.172,381.324z"/>
<circle style="fill:#FFFFFF;" cx="161.83" cy="419.861" r="4.056"/>
<path style="fill:#9B7E73;" d="M441.838,309.999v44.636c0,20.836,7.903,40.896,22.115,56.132l0,0
	c3.616,3.876,1.921,10.2-3.149,11.749l0,0c-74.931,22.884-154.986,22.884-229.917,0l0,0c-5.07-1.548-6.765-7.872-3.149-11.749l0,0
	c14.212-15.236,22.115-35.296,22.115-56.132v-44.636H441.838z"/>
<path style="fill:#81D4AE;" d="M410.71,447.794c0-35.824-29.041-64.865-64.865-64.865l0,0c-35.824,0-64.865,29.041-64.865,64.865
	H410.71z"/>
<path style="fill:#FFFFFF;" d="M345.846,434.576c-11.183,0-21.354-5.687-27.209-15.213l-12.969-21.097
	c-2.169-3.529-1.066-8.148,2.462-10.317c3.527-2.169,8.146-1.068,10.317,2.461l12.969,21.098c3.106,5.052,8.499,8.068,14.43,8.068
	c5.93,0,11.323-3.016,14.429-8.067l12.968-21.098c2.169-3.53,6.788-4.631,10.317-2.462c3.528,2.169,4.631,6.788,2.462,10.317
	l-12.968,21.098C367.198,428.889,357.027,434.576,345.846,434.576z"/>
<path style="opacity:0.1;enable-background:new    ;" d="M345.845,382.929L345.845,382.929c-18.59,0-35.35,7.822-47.177,20.352
	c13.933,7.879,30.028,12.38,47.177,12.38c17.148,0,33.244-4.502,47.177-12.38C381.195,390.751,364.435,382.929,345.845,382.929z"/>
<circle style="fill:#FFD4B8;" cx="249.85" cy="310.121" r="16.929"/>
<circle style="opacity:0.1;fill:#513130;enable-background:new    ;" cx="249.85" cy="310.121" r="16.929"/>
<g>
	<circle style="fill:#FFD4B8;" cx="441.84" cy="310.121" r="16.929"/>
	<circle style="fill:#FFD4B8;" cx="345.85" cy="310.001" r="95.99"/>
</g>
<path style="fill:#9B7E73;" d="M345.845,214.006c-50.956,0-92.63,39.705-95.792,89.863c38.254,2.772,83.222-46.491,85.629-61.847
	c1.9,19.625,72.285,68.668,105.861,60.427C437.693,252.963,396.319,214.006,345.845,214.006z"/>
<g>
	<path style="fill:#4A5058;" d="M345.845,348.782c-5.899,0-11.598-2.472-15.636-6.782c-2.832-3.023-2.677-7.769,0.346-10.601
		c3.024-2.832,7.77-2.676,10.602,0.346c1.23,1.314,2.895,2.037,4.688,2.037c1.794,0,3.459-0.723,4.69-2.037
		c2.832-3.022,7.577-3.178,10.602-0.346c3.022,2.832,3.178,7.578,0.346,10.601C357.443,346.31,351.744,348.782,345.845,348.782z"/>
	<path style="fill:#4A5058;" d="M305.841,322.135c-4.143,0-7.5-3.358-7.5-7.5c0-0.646-0.525-1.171-1.171-1.171
		c-0.645,0-1.171,0.525-1.171,1.171c0,4.142-3.357,7.5-7.5,7.5s-7.5-3.358-7.5-7.5c0-8.917,7.254-16.171,16.171-16.171
		s16.171,7.254,16.171,16.171C313.341,318.777,309.983,322.135,305.841,322.135z"/>
	<path style="fill:#4A5058;" d="M403.191,322.135c-4.143,0-7.5-3.358-7.5-7.5c0-0.646-0.525-1.171-1.171-1.171
		s-1.171,0.525-1.171,1.171c0,4.142-3.357,7.5-7.5,7.5s-7.5-3.358-7.5-7.5c0-8.917,7.254-16.171,16.171-16.171
		s16.171,7.254,16.171,16.171C410.691,318.777,407.334,322.135,403.191,322.135z"/>
</g>
<path style="opacity:0.1;fill:#513130;enable-background:new    ;" d="M360.071,393.873c-53.015,0-95.992-42.977-95.992-95.993
	c0-25.912,10.271-49.422,26.958-66.692c-24.892,17.343-41.184,46.172-41.184,78.81c0,53.015,42.977,95.992,95.993,95.992
	c27.103,0,51.579-11.237,69.034-29.301C399.339,387.519,380.448,393.873,360.071,393.873z"/>
<path style="fill:#FBAB5F;" d="M461.522,454.862H50.478C22.644,454.862,0,432.218,0,404.384V189.442
	c0-27.833,22.644-50.477,50.478-50.477h411.045c27.834,0,50.478,22.644,50.478,50.477v214.943
	C512,432.218,489.356,454.862,461.522,454.862z M50.478,169.797c-10.833,0-19.645,8.813-19.645,19.645v214.943
	c0,10.832,8.812,19.645,19.645,19.645h411.045c10.833,0,19.645-8.813,19.645-19.645V189.442c0-10.832-8.812-19.645-19.645-19.645
	L50.478,169.797L50.478,169.797z"/>
<path style="opacity:0.1;enable-background:new    ;" d="M91.984,138.964H50.477C22.644,138.964,0,161.609,0,189.442v214.943
	c0,27.833,22.644,50.477,50.478,50.477h121.559C107.047,400.925,48.787,287.236,91.984,138.964z"/>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
</svg>
