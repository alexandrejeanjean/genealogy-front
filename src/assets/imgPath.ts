const add = require("./add.svg");
const deletePic = require("./delete.svg");
const addYellow = require("./add-yellow.svg");
const father = require("./avatars/father.svg");
const mother = require("./avatars/mother.svg");
const grandFather = require("./avatars/grandfather.svg");
const grandMother = require("./avatars/grandmother.svg");
const son = require("./avatars/son.svg");
const daughter = require("./avatars/daughter.svg");
const baby = require("./avatars/baby.svg");
const family = require("./family.svg");
const tree = require("./tree.svg");

export {
  add,
  deletePic,
  addYellow,
  father,
  mother,
  grandFather,
  grandMother,
  son,
  daughter,
  baby,
  family,
  tree,
};
