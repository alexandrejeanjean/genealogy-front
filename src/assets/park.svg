<?xml version="1.0" encoding="iso-8859-1"?>
<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
<g>
	<path style="fill:#5EC08B;" d="M498.299,475.375c7.895,0,13.772-7.314,12.058-15.021c-6.854-30.811-34.35-53.847-67.225-53.847
		c-38.035,0-68.868,30.833-68.868,68.868H498.299z"/>
	<circle style="fill:#5EC08B;" cx="284.5" cy="159.92" r="123.3"/>
</g>
<g>
	<rect x="130.15" y="394.74" style="fill:#81D4AE;" width="262.66" height="80.64"/>
	<path style="fill:#81D4AE;" d="M199.337,103.801C110.05,103.802,37.669,176.183,37.669,265.47c0,0.015,0,0.03,0,0.046
		c0.001,2.823-1.176,5.496-3.233,7.429c-15.106,14.201-24.347,34.568-23.686,57.086c1.144,38.958,32.395,70.905,71.319,72.884
		c9.377,0.477,18.393-0.781,26.778-3.456c25.822,17.474,56.964,27.68,90.491,27.68c89.287,0,161.668-72.381,161.669-161.667
		C361.008,176.268,288.541,103.801,199.337,103.801z"/>
</g>
<g>
	<path style="fill:#5EC08B;" d="M143.745,225.257c8.487,8.487,11.723,20.408,12.932,29.171c0.879,6.369-4.58,11.828-10.949,10.949
		c-8.762-1.209-20.684-4.445-29.171-12.932s-11.723-20.409-12.932-29.171c-0.879-6.369,4.58-11.828,10.949-10.949
		C123.337,213.535,135.259,216.77,143.745,225.257z"/>
	<path style="fill:#5EC08B;" d="M229.414,176.878c8.487-8.487,20.408-11.723,29.171-12.932c6.369-0.879,11.828,4.58,10.949,10.949
		c-1.209,8.762-4.445,20.684-12.932,29.171c-8.487,8.487-20.409,11.723-29.171,12.932c-6.369,0.879-11.828-4.58-10.949-10.949
		C217.692,197.287,220.927,185.365,229.414,176.878z"/>
</g>
<path style="opacity:0.05;enable-background:new    ;" d="M99.788,348.712c-27.678,0-53.281,8.88-74.12,23.943
	c13.01,17.412,33.36,29.088,56.402,30.26c9.377,0.477,18.394-0.781,26.778-3.456c25.822,17.474,56.964,27.68,90.491,27.68
	c5.818,0,11.562-0.314,17.222-0.914C197.367,380.683,152.312,348.712,99.788,348.712z"/>
<path style="fill:#5EC08B;" d="M252.676,475.375c0-69.954-56.709-126.663-126.663-126.663c-65.363,0-119.162,49.509-125.943,113.072
	c-0.775,7.268,4.976,13.591,12.285,13.591H252.676z"/>
<path style="opacity:0.1;enable-background:new    ;" d="M252.676,475.375c0-69.954-56.709-126.663-126.663-126.663
	c-65.363,0-119.162,49.509-125.943,113.072c-0.775,7.268,4.976,13.591,12.285,13.591H252.676z"/>
<rect x="395.51" y="317.49" style="fill:#9B7E73;" width="61.842" height="157.88"/>
<path style="opacity:0.1;enable-background:new    ;" d="M381.652,83.994c-33.659,26.288-59.812,118.274-65.549,195.127
	C368.886,265.166,407.8,217.09,407.8,159.92C407.799,131.281,398.033,104.924,381.652,83.994z"/>
<path style="fill:#81D4AE;" d="M426.434,76.643C473.691,76.643,512,215.422,512,309.17c0,47.257-38.309,85.566-85.566,85.566
	s-85.566-38.309-85.566-85.566C340.868,215.422,379.177,76.643,426.434,76.643z"/>
<g>
	<path style="fill:#FFFFFF;" d="M397.933,256.663H125.028c-7.897,0-14.299,6.402-14.299,14.299v32.767
		c0,7.897,6.402,14.299,14.299,14.299h272.905c7.897,0,14.299-6.402,14.299-14.299v-32.767
		C412.232,263.065,405.83,256.663,397.933,256.663z"/>
	<path style="fill:#FFFFFF;" d="M397.933,318.029H125.028c-7.897,0-14.299,6.402-14.299,14.299v32.767
		c0,7.897,6.402,14.299,14.299,14.299h272.905c7.897,0,14.299-6.402,14.299-14.299v-32.767
		C412.232,324.431,405.83,318.029,397.933,318.029z"/>
</g>
<path style="opacity:0.1;enable-background:new    ;" d="M397.933,318.029H125.028c-7.897,0-14.299,6.402-14.299,14.299v32.767
	c0,7.897,6.402,14.299,14.299,14.299h272.905c7.897,0,14.299-6.402,14.299-14.299v-32.767
	C412.232,324.431,405.83,318.029,397.933,318.029z"/>
<path style="opacity:0.1;enable-background:new    ;" d="M133.752,365.095v-32.767c0-7.897,6.402-14.299,14.299-14.299h-23.024
	c-7.897,0-14.299,6.402-14.299,14.299v32.767c0,7.897,6.402,14.299,14.299,14.299h23.024
	C140.154,379.394,133.752,372.992,133.752,365.095z"/>
<path style="opacity:0.1;enable-background:new    ;" d="M133.752,303.73v-32.767c0-7.897,6.402-14.299,14.299-14.299h-23.024
	c-7.897,0-14.299,6.402-14.299,14.299v32.767c0,7.897,6.402,14.299,14.299,14.299h23.024
	C140.154,318.029,133.752,311.627,133.752,303.73z"/>
<path style="fill:#FFFFFF;" d="M433.464,379.394H89.497c-4.057,0-7.347,3.289-7.347,7.346v15.99c0,4.057,3.289,7.346,7.347,7.346
	h343.967c4.057,0,7.346-3.289,7.346-7.346v-15.99C440.81,382.683,437.521,379.394,433.464,379.394z"/>
<path style="opacity:0.1;enable-background:new    ;" d="M110.729,402.73v-15.99c0-4.057,3.289-7.346,7.346-7.346H89.497
	c-4.057,0-7.347,3.289-7.347,7.346v15.99c0,4.057,3.289,7.346,7.347,7.346h28.578C114.018,410.077,110.729,406.788,110.729,402.73z"
	/>
<path style="fill:#FFFFFF;" d="M140.43,475.375h-20.557c-3.383,0-6.125-2.742-6.125-6.125v-59.173h32.807v59.173
	C146.555,472.633,143.813,475.375,140.43,475.375z"/>
<rect x="113.75" y="410.08" style="opacity:0.15;enable-background:new    ;" width="32.807" height="17.06"/>
<path style="fill:#FFFFFF;" d="M382.531,475.375h20.557c3.383,0,6.125-2.742,6.125-6.125v-59.173h-32.807v59.173
	C376.406,472.633,379.148,475.375,382.531,475.375z"/>
<rect x="376.41" y="410.08" style="opacity:0.15;enable-background:new    ;" width="32.807" height="17.06"/>
<g>
	<circle style="fill:#4A5058;" cx="139.56" cy="348.71" r="7.338"/>
	<circle style="fill:#4A5058;" cx="383.41" cy="348.71" r="7.338"/>
	<circle style="fill:#4A5058;" cx="139.56" cy="287.35" r="7.338"/>
	<circle style="fill:#4A5058;" cx="383.41" cy="287.35" r="7.338"/>
</g>
<path style="fill:#5EC08B;" d="M425.258,206.866c-8.487-8.487-11.723-20.409-12.932-29.171c-0.879-6.369,4.58-11.828,10.949-10.949
	c8.762,1.209,20.684,4.445,29.171,12.932s11.723,20.409,12.932,29.171c0.879,6.369-4.58,11.828-10.949,10.949
	C445.666,218.588,433.745,215.353,425.258,206.866z"/>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
</svg>
