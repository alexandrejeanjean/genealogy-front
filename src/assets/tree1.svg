<svg id="Capa_1" enable-background="new 0 0 512 512" height="512" viewBox="0 0 512 512" width="512" xmlns="http://www.w3.org/2000/svg"><g><g><path d="m226.097 408.481c-7.744 45.362-22.343 73.238-33.022 88.46-4.449 6.343.074 15.059 7.822 15.059h110.207c7.748 0 12.271-8.716 7.822-15.059-10.679-15.222-25.279-43.098-33.022-88.46l-29.903-9z" fill="#cf8151"/><path d="m226.088 408.48c9.238 3.741 19.331 5.809 29.912 5.809 10.58 0 20.674-2.068 29.912-5.809-.956-5.602-1.815-11.455-2.542-17.599l-27.37-9.002-27.37 8.999c-.727 6.146-1.585 11.999-2.542 17.602z" fill="#bd6e45"/><path d="m492.867 206.691c0-38.317-27.01-70.313-63.026-78.02.013-.594.045-1.184.045-1.781 0-44.073-35.728-79.802-79.802-79.802-6.975 0-13.739.901-20.188 2.582-11.89-29.132-40.488-49.67-73.896-49.67s-62.006 20.538-73.897 49.669c-6.449-1.681-13.213-2.582-20.188-2.582-44.073 0-79.802 35.728-79.802 79.802 0 .597.032 1.187.045 1.781-36.016 7.708-63.026 39.704-63.026 78.02 0 38.317 27.01 70.313 63.026 78.02-.013.594-.045 1.184-.045 1.781 0 6.629.822 13.063 2.346 19.221 16.217 25.005 47.06 49.2 77.456 49.2 8.658 0 16.989-1.391 24.794-3.944 2.099-.687 4.388.027 5.724 1.785 14.574 19.181 37.619 31.573 63.567 31.573s48.993-12.392 63.567-31.573c1.336-1.758 3.625-2.472 5.724-1.785 7.805 2.553 16.136 3.944 24.794 3.944 30.396 0 61.239-24.195 77.456-49.2 1.523-6.158 2.345-12.592 2.345-19.22 0-.597-.032-1.187-.045-1.781 36.016-7.708 63.026-39.704 63.026-78.02z" fill="#b3e59f"/><g><g><g><path d="m350.085 347.713c-8.658 0-16.989-1.391-24.794-3.944-2.099-.687-4.388.027-5.724 1.786-14.574 19.181-37.619 31.573-63.567 31.573s-48.993-12.392-63.567-31.573c-1.336-1.758-3.625-2.472-5.724-1.786-7.805 2.553-16.136 3.944-24.794 3.944-30.396 0-61.239-16.995-77.456-42 8.604 34.786 40.012 60.581 77.456 60.581 9.977 0 19.519-1.847 28.323-5.191 14.39 20.897 38.47 34.605 65.762 34.605s51.372-13.708 65.762-34.605c8.804 3.344 18.346 5.191 28.323 5.191 37.445 0 68.852-25.795 77.456-60.581-16.217 25.005-47.06 42-77.456 42z" fill="#95d6a4"/></g></g></g><path d="m362.518 204.021h-98.776v-14.885h42.292c4.276 0 7.742-3.466 7.742-7.742v-36.551c0-4.276-3.466-7.742-7.742-7.742s-7.742 3.466-7.742 7.742v28.81h-84.585v-28.81c0-4.276-3.466-7.742-7.742-7.742s-7.742 3.466-7.742 7.742v36.551c0 4.276 3.466 7.742 7.742 7.742h42.292v14.885h-98.776c-4.276 0-7.742 3.466-7.742 7.742v37.232c0 4.276 3.466 7.742 7.742 7.742s7.742-3.466 7.742-7.742v-29.49h91.034v29.49c0 4.276 3.466 7.742 7.742 7.742s7.742-3.466 7.742-7.742v-29.49h91.034v29.49c0 4.276 3.466 7.742 7.742 7.742s7.742-3.466 7.742-7.742v-37.232c.001-4.276-3.466-7.742-7.741-7.742z" fill="#f6e080"/><path d="m149.482 291.399c13.682 0 29.2-18.09 29.2-30.347-3.935-12.378-15.518-21.348-29.2-21.348-13.681 0-25.265 8.969-29.2 21.348.001 12.685 15.519 30.347 29.2 30.347z" fill="#dc4955"/><path d="m149.482 282.399c-13.681 0-25.265-8.97-29.2-21.348-.932 2.931-1.438 6.051-1.438 9.29 0 16.921 13.717 30.638 30.638 30.638s30.638-13.717 30.638-30.638c0-3.239-.507-6.36-1.438-9.29-3.935 12.379-15.518 21.348-29.2 21.348z" fill="#d82f3c"/><path d="m256 291.399c13.682 0 29.2-18.09 29.2-30.347-3.935-12.378-15.518-21.348-29.2-21.348-13.681 0-25.265 8.969-29.2 21.348 0 12.685 15.518 30.347 29.2 30.347z" fill="#dc4955"/><path d="m256 282.399c-13.681 0-25.265-8.97-29.2-21.348-.932 2.931-1.438 6.051-1.438 9.29 0 16.921 13.717 30.638 30.638 30.638s30.638-13.717 30.638-30.638c0-3.239-.506-6.36-1.438-9.29-3.935 12.379-15.519 21.348-29.2 21.348z" fill="#d82f3c"/><path d="m362.518 291.399c13.681 0 29.2-18.09 29.2-30.347-3.935-12.378-15.518-21.348-29.2-21.348s-25.265 8.969-29.2 21.348c0 12.685 15.518 30.347 29.2 30.347z" fill="#dc4955"/><path d="m362.518 282.399c-13.682 0-25.265-8.97-29.2-21.348-.932 2.931-1.438 6.051-1.438 9.29 0 16.921 13.717 30.638 30.638 30.638s30.638-13.717 30.638-30.638c0-3.239-.506-6.36-1.438-9.29-3.936 12.379-15.519 21.348-29.2 21.348z" fill="#d82f3c"/><path d="m306.034 144.552c13.682 0 29.2-18.09 29.2-30.347-3.935-12.378-15.518-21.348-29.2-21.348-13.681 0-25.265 8.969-29.2 21.348.001 12.685 15.519 30.347 29.2 30.347z" fill="#dc4955"/><path d="m306.034 135.552c-13.681 0-25.265-8.97-29.2-21.348-.932 2.931-1.438 6.051-1.438 9.29 0 16.921 13.717 30.638 30.638 30.638s30.638-13.717 30.638-30.638c0-3.239-.506-6.36-1.438-9.29-3.935 12.378-15.518 21.348-29.2 21.348z" fill="#d82f3c"/><path d="m205.965 144.552c13.682 0 29.2-18.09 29.2-30.347-3.935-12.378-15.518-21.348-29.2-21.348-13.681 0-25.265 8.969-29.2 21.348.001 12.685 15.519 30.347 29.2 30.347z" fill="#dc4955"/><path d="m205.965 135.552c-13.681 0-25.265-8.97-29.2-21.348-.932 2.931-1.438 6.051-1.438 9.29 0 16.921 13.717 30.638 30.638 30.638s30.638-13.717 30.638-30.638c0-3.239-.507-6.36-1.438-9.29-3.935 12.378-15.518 21.348-29.2 21.348z" fill="#d82f3c"/></g></g></svg>