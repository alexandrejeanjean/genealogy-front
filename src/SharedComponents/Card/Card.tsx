import React from "react";
import { deletePic } from "../../assets/imgPath";
import "./card.scss";

type TCard = {
  text: string;
  subtext?: string;
  picture: any;
  handleClick: () => any;
  handleDelete: () => any;
};

const Card = ({ text, subtext, handleClick, handleDelete, picture }: TCard) => {
  return (
    <div className="card-wrapper">
      <button className="card" onClick={() => handleClick()}>
        <img src={picture} alt="" />
        <p className="card-text">{text}</p>
        <p className="card-subtext"> {subtext}</p>
      </button>
      <button className="delete-btn" onClick={() => handleDelete()}>
        <img src={deletePic} alt="delete button" />
      </button>
    </div>
  );
};

export default Card;
