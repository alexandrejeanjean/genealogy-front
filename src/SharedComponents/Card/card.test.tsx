import React from "react";
import { shallow, mount } from "enzyme";
import Card from "./Card";
import { family } from "../../assets/imgPath";

const CardProps = {
  text: "Family 1",
  picture: family,
  handleClick: jest.fn(),
  handleDelete: jest.fn(),
};

describe("Card", () => {
  it("should render correctly", () => {
    const Component = shallow(<Card {...CardProps} />);
    expect(Component).toMatchSnapshot();
  });
});
