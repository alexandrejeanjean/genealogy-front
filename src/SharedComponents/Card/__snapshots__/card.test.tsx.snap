// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render correctly 1`] = `
<div
  className="card-wrapper"
>
  <button
    className="card"
    onClick={[Function]}
  >
    <img
      alt=""
      src={[Function]}
    />
    <p
      className="card-text"
    >
      Family 1
    </p>
    <p
      className="card-subtext"
    >
       
    </p>
  </button>
  <button
    className="delete-btn"
    onClick={[Function]}
  >
    <img
      alt="delete button"
      src={[Function]}
    />
  </button>
</div>
`;
